later/lib/database/models/record_model.dart

```diff
@@ -3,23 +3,23 @@ import 'package:isar/isar.dart';
 part 'record_model.g.dart';
 
 @collection
-class RecordModel {
-  RecordModel({
-    this.title,
+class RecordDbModel {
+  RecordDbModel({
+    required this.title,
     this.description,
     required this.url,
     required this.lastEditedAt,
     required this.createdAt,
   });
 
-  String url;
+  String? url;
   String? description;
   DateTime createdAt;
-  DateTime lastEditedAt;
+  DateTime? lastEditedAt;
   Id id = Isar.autoIncrement; // you can also use id = null to auto increment
 
   @Index(type: IndexType.value)
-  String? title;
+  String title;
```

Comment:
Let's reorder this a bit.

1. For ID, it is a convention to be the first.
2. Let's put prioritised things first as well. So title goes second.
3. Others go in the prioritised order as well. Like `URL —> Description —> CreatedAt —> LastEditedAt`.